### PR TITLE
Updated Metric data model

### PR DESCRIPTION
**Pull request by robhudson**
This adds a few extra columns required to support the upcoming Spark
script that will aggregate and import data.

### CODE REVIEW

File: distributionviewer/api/models.py [hunk 1]
```diff
@@ -6,6 +6,12 @@
 class Metric(models.Model):
     name = models.CharField(max_length=255, unique=True)
     description = models.TextField()
+    type = models.CharField(
+        max_length=1, choices=(('C', 'Categorical'), ('N', 'Numerical')),
```
Comment (spasovski) [inline distributionviewer/api/models.py:10]: Will this mean 'type' is now a single char?

Comment (spasovski) [inline distributionviewer/api/models.py:10]: n/m I see these are just unexposed additions to the base model

Comment (robhudson) [inline distributionviewer/api/models.py:10]: Yeah, I just needed this for the other side when aggregating the data. If they are useful for the front-end we can add them to the API.

**Comment by spasovski** [comment]
r+ from me if you're in a hurry. Might be worthwhile to have this checked by @openjck also though.

**Comment by openjck** [comment]
Hey all, just saw this upon getting back from PTO. FWIW this looks good to me. Glad to see it's been merged.

**Pull request merged**